packages/demo/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "demo",
-  "version": "2.0.9",
+  "version": "2.1.0",
```

No need to bump this for the PR
```suggestion
  "version": "2.0.9",
```